gef.py

```diff
@@ -5179,12 +5180,36 @@ def do_invoke(self, argv):
 
         pattern = argv[0]
         endian = get_endian()
-        if argc==2:
-            if argv[1]=="big": endian = Elf.BIG_ENDIAN
-            elif argv[1]=="small": endian = Elf.LITTLE_ENDIAN
 
-        info("Searching '{:s}' in memory".format(Color.yellowify(pattern)))
-        self.search_pattern(pattern, endian)
+        if argc >= 2:
+            if argv[1] == "big": endian = Elf.BIG_ENDIAN
```

Comment:
Replace with `argv[1].lower()` just in case. Doesn't cost much and avoid potential Issues